specifications/api/swagger.yaml

```diff
@@ -2065,7 +2065,7 @@ paths:
 
   /byron-wallets/{walletId}/statistics/utxos:
     get:
-      operationId: getUTxOsStatistics
+      operationId: getUTxOsStatisticsByron
```

Comment:
then you need to update https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/Api.hs#L401